include/deal.II/base/quadrature_lib.h

```diff
@@ -647,6 +647,231 @@ class QGaussLobattoChebyshev : public Quadrature<dim>
 
 };
 
+/**
+ * Given an arbitrary quadrature formula, return one that chops the quadrature
+ * points above the hyper-plane defined by $\sum_i x_i = 1$. In other words,
+ * it extracts those quadrature points from the base formula that satisfy
+ * $\sum_i (\mathbf x_q)_i \le 1+10^{-12}$."
+ *
+ * In general the resulting quadrature is not very useful, unless the
+ * quadrature you started from has been constructed specifically to integrate
+ * over triangles or tetrahedra. This class only ensures that the resulting
+ * quadrature formula only has quadrature points in the reference simplex or on
+ * its boundary.
+ *
+ * No transformation is applied to the weights, and the weights referring to points
+ * that live outside the reference simplex are simply discarded.
+ *
+ * The main use of this quadrature formula is not to chop tensor product
+ * quadratures. Ideally you should pass to this class a quadrature formula
+ * constructed directly using points and weights in the reference simplex,
+ * capable of integrating on triangles or tetrahedra.
+ *
+ * For finite elements based on quadrilaterals and hexahedra, a QSimplex
+ * quadrature formula is not very useful on its own. This class is typically
+ * used in conjuction with other classes, like QSplit, to patch the reference
+ * element using several QSimplex quadrature formulas.
+ *
+ * Such quadrature formulas are useful to integrate functions with
+ * singularities at certain points, or functions that present jumps along a
+ * co-dimension one surface inside the reference element, like in the extended
+ * finite element method.
```

Comment:
Maybe add abbreviation

> extended finite element method (XFEM).

just in case someone will be google XFEM in documentation